src/main/java/frc/robot/subsystems/shooter/ShooterBedIO.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.shooter;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface ShooterBedIO
+{
+    @AutoLog
+    public static class ShooterBedIOInputs
+    {
+        public double     bedLeaderAppliedVolts   = 0.0;
+        public double[]   bedLeaderCurrentAmps    = new double[] {};
+        public double     bedFollowerAppliedVolts = 0.0;
+        public double[]   bedFollowerCurrentAmps  = new double[] {};
```

Comment:
We don't really have a way to measure the current from the VictorSPX's directly. Let's remove the current inputs:

```suggestion
        public double     bedLeaderAppliedVolts   = 0.0;
        public double     bedFollowerAppliedVolts = 0.0;
```